.github/workflows/image-push.yml

```
@@ -0,0 +1,31 @@
+on:
+  release:
+    types: [published]
+  
```

Set least privileges to the GH token:

```suggestion

permissions: 
  packages: write # publish images to ghcr.io/<repo>.
```